### ISSUE TITLE
Number of recovered cases in all US counties are 0, and do not match with the state gov websites.

### ISSUE THREAD

**Issue by anannyanath7** [CSSEGISandData/COVID-19#2339]

**Comment by weathermatrix** [comment]
Yes all zeroes, it would be nice if it was not available they could N/A it.

**Comment by gcamp-hearst** [comment]
Has anyone seen any updates in the feed?

**Comment by markusMM** [comment]
Hi,
below are the "Province/State" which do not seem to be in the "Recovered" dataset:
'Canada - Alberta' 'Canada - British Columbia' 'Canada - Manitoba'
 'Canada - New Brunswick' 'Canada - Newfoundland and Labrador'
 'Canada - Northwest Territories' 'Canada - Nova Scotia'
 'Canada - Ontario' 'Canada - Prince Edward Island' 'Canada - Quebec'
 'Canada - Saskatchewan' 'Canada - Yukon' 'US - Alabama' 'US - Alaska'
 'US - Arizona' 'US - Arkansas' 'US - California' 'US - Colorado'
 'US - Connecticut' 'US - Delaware' 'US - District of Columbia'
 'US - Florida' 'US - Georgia' 'US - Guam' 'US - Hawaii' 'US - Idaho'
 'US - Illinois' 'US - Indiana' 'US - Iowa' 'US - Kansas' 'US - Kentucky'
 'US - Louisiana' 'US - Maine' 'US - Maryland' 'US - Massachusetts'
 'US - Michigan' 'US - Minnesota' 'US - Mississippi' 'US - Missouri'
 'US - Montana' 'US - Nebraska' 'US - Nevada' 'US - New Hampshire'
 'US - New Jersey' 'US - New Mexico' 'US - New York' 'US - North Carolina'
 'US - North Dakota' 'US - Ohio' 'US - Oklahoma' 'US - Oregon'
 'US - Pennsylvania' 'US - Puerto Rico' 'US - Rhode Island'
 'US - South Carolina' 'US - South Dakota' 'US - Tennessee' 'US - Texas'
 'US - Utah' 'US - Vermont' 'US - Virgin Islands' 'US - Virginia'
 'US - Washington' 'US - West Virginia' 'US - Wisconsin' 'US - Wyoming'

They are 66 in total!

I have looked over the "Recovered" set and saw a few generalizations.
Below are those ones which does not seem to appear in the "Confirmed" or "Deaths" sets:

0    Canada
1        US
2     Yemen
dtype: object
length: 3

If there is a way to bind more specific information about "Province/State" Canada and US, it would be more than helpful.

-M

**Comment by ryanewtaylor** [comment]
Why are the recovered values not reported. This would help to calculate active cases, would it not, with 

> Active = Confirmed - Recovered - Deaths
